README.md

```diff
@@ -381,6 +381,55 @@ const hubspot = new Hubspot(params)
 return hubspot.oauth.getAccessToken(params).then(...)
 ```
 
+## Contributing
+
+Before submitting a pull request, please make sure the following is done:
+
+1. Fork the repository and create your branch from master.
+1. Run npm install in the repository root.
+1. If you’ve fixed a bug or added code that should be tested, add tests!
+1. Ensure the test suite passes (npm test). Tip: `npm run mocha -- --grep "test name"` is helpful in development.
+1. Format your code with prettier and eslint (npm run lint).
+
+## Development Workflow
+
+After cloning this repo, run `npm install` to fetch dependencies. Then run the
+test suite with `npm test`. From master, this should be green. Our tests mock
+out requests to HubSpot's api using [nock]. We then recommend running the test
+suite without mocking api requests with `NOCK_OFF=true npm run mocha`. This
+should fail as you'll need some [environment variables] for real requests to
+pass.
+
+[nock]: https://github.com/nock/nock
+[environment variables]: https://github.com/motdotla/dotenv
+
+If you haven't already, create a [developer account] on hubspot. You'll want to
+[create an app] as well. Then, create a new file, `.env` in the root of the
```

Comment:
I've found a `.env.template` helpful in past projects. Then you can have the environment variable keys (but no values) present and add `.env` to `.gitignore`.

Reply:
Added in 
https://github.com/MadKudu/node-hubspot/pull/138/commits/945bf27685ddb34f88f5a6a8b48944a20da00320